middlewares/accesslog/json_log_formatter.go

```diff
@@ -0,0 +1,235 @@
+package accesslog
```

Comment:
Just to be sure, is this code from you or did you copied it from another place (just want to avoid license issue ;) )?